qiskit/aqua/operators/converters/abelian_grouper.py

```diff
@@ -88,30 +87,33 @@ def group_subops(cls, list_op: ListOp, fast: bool = True, use_nx: bool = False)
 
         Args:
             list_op: The Operator to group into Abelian groups
-            fast: Enable the fast commutation graph generation if all operators are Pauli operators
-            use_nx: Enable networkx.coloring.greedy_color instead of the numpy-based coloring
+            fast: DEPRECATED. Enable the fast commutation graph generation if all operators are
```

Comment:
Since neither of these 2 params do anything now presumably the description could just be `Ignored - parameter will be  removed in future release`

Reply:
I did it too.